STEP_Project/OUListContent.js

```diff
@@ -7,6 +7,27 @@ function addRemoveButtonClick (event)
     if (event.target.getAttribute("class")!=='bClass') return;
 }
 
+/**
+ * Adds a button to every visible row when the page is first loaded.
+ * @param {HTML Object} tabl - contains OU rows
+ */
+function initButtons(tabl)
+{
+    let OUs = tabl.rows;
```

Comment:
nit: Rename this to orgUnits, here and throughout (in tests as well). Similarly for other places like numOUs etc.

Reply:
Made all these changes.